### ISSUE TITLE
Text between magic comments is not ignored when using --fix

### ISSUE THREAD

**Issue by orangain** [textlint/textlint-filter-rule-comments#1]
## Environment

* textlint@6.8.0
* [textlint-plugin-review](https://github.com/orangain/textlint-plugin-review)@0.3.1
* textlint-rule-ignore-comments@1.1.0
* [textlint-rule-prh](https://github.com/azu/textlint-rule-prh)@3.0.1

## Problem

Using textlint-rule-ignore-comments, a text between magic comments, `textlint-disable` and `textlint-enable`, is **ignored** as expected. However, when using --fix, the text between magic comments is **fixed and not ignored**. This behavior is unexpected to me.

## How to reproduce

Install dependencies:

```
$ npm install textlint textlint-plugin-review textlint-rule-ignore-comments textlint-rule-prh
```

Create `test.re`:

```
#@# textlint-disable prh
「known hostsリストにサーバ"127.0.0.1"のエントリはありません」というダイアログが表示された場合は、「このホストをknown_hostsリストに追加する」にチェックを入れて「続行」をクリックします。
#@# textlint-enable prh
```

Create `.textlintrc`:

```
plugins:
  - "review"
rules:
  ignore-comments: true
  prh:
    rulePaths:
      - "prh.yml"
```

Create `prh.yml`:

```
version: 1
rules:
  - expected: サーバー
    pattern: /サーバ(?!ー)/
```

When using lint, a text between magic comments is **ignored** as expected.

```
$ $(npm bin)/textlint --experimental test.re
```

When using fix, the text between magic comments is **fixed and not ignored** unexpectedly.

```
$ $(npm bin)/textlint --experimental --fix test.re

/Users/orange/samplecodes/ignore-textlint-fix/test.re
  2:17  ✔   サーバ => サーバー  prh

✔ Fixed 1 problem
```

## My use case

I'm using the prh rule to fix variety of spelling. I'd like to automatically fix many problems reported by the rule except for only one sentence. The sentence I want to ignore `「known hostsリストにサーバ"127.0.0.1"のエントリはありません」` is sort of like an inline quote. As there is no appropriate node type for the inline quote both in Re:VIEW and textlint, the textlint-rule-ignore-comments really sounds good to me.

**Comment by azu** [comment]
Ah, nice catch.
It is a bug.

**Comment by azu** [comment]
It is a bug of textlint and ignore rule.
I've filed issue to textlint https://github.com/textlint/textlint/issues/211

**Comment by azu** [comment]
This issue is resolved by textlint [6.9.0](https://github.com/textlint/textlint/releases/tag/6.9.0 "6.9.0").
And update README.md of textlint-filter-rule-comments/
